web/components/project/dropdowns/filters/created-at.tsx

```diff
@@ -24,6 +24,17 @@ export const FilterCreatedDate: React.FC<Props> = observer((props) => {
     d.name.toLowerCase().includes(searchQuery.toLowerCase())
   );
 
+  const isCustomDateSelected = () => {
+    const isCustomFateApplied = appliedFilters?.filter((f) => f.includes("-")) || [];
```

Comment:
This condition seems to generic, please verify if the string is a valid date instead